rocketpy/control/controller.py

```diff
@@ -84,8 +84,10 @@ def __init__(
         None
         """
         self.interactive_objects = interactive_objects
+        self.base_controller_function = controller_function
```

Comment:
@MateusStano I have no idea why you want to save this attribute as `base_controller_function`instead of simply `controller_function`. I'm approving this since the class is still private.